vicodyn/src/vicodyn/proxy.cpp

```diff
@@ -101,25 +110,45 @@ class safe_stream_t {
 
 class vicodyn_dispatch_t : public std::enable_shared_from_this<vicodyn_dispatch_t> {
     using protocol = io::protocol<app_tag>::scope;
+    using clock_t = peers_t::clock_t;
+
+    struct event_t {
+        std::string frame;
```

Comment:
Why aren't `frame` and `headers` in `buffer_t`?

Reply:
Fixed